CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+When contributing to this service,
+please open an issue on
+[GitHub issues](https://github.com/onOfficeGmbH/sdk/issues)
+or send a mail to `apisupport@onoffice.de`
+this service before making a change.
+
+## Bugs
+
+Bugs must be reported via the
+[GitHub issues](https://github.com/onOfficeGmbH/sdk/issues)
+Bug fixes and improvements can be provided via a Pull Request.
+
+A fix that redefines that behaviour of an implementation are considered
+as a [breaking change](#breaking-changes).
```

Comment:
Maybe some like "A fix that introduces a change in behavior is considered a breaking change"?